.github/workflows/require-checklist.yml

```diff
@@ -0,0 +1,23 @@
+name: Require Reviewer Checklist
+
+on:
+  pull_request:
+    types: [opened, reopened, edited, synchronize]
+    branches: [master]
```

Comment:
I believe we also want to target branches starting with `1.` like now 1.40.x for dev/beta and 1.39.x for release channel